### PR TITLE
Generate application-utility package

### PR DESCRIPTION
**Pull request by yuderekyu**
### Summary
Initial package for application-utility. Will house #29 and #34.

Thanks for contributing to Terra.
@cerner/terra

[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md
[License]: ../blob/master/LICENSE

### CODE REVIEW

File: packages/terra-application-utility/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Terra Application Utility
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-application-utility.svg)](https://www.npmjs.org/package/terra-application-utility)
+[![Build Status](https://travis-ci.org/cerner/terra-framework.svg?branch=master)](https://travis-ci.org/cerner/terra-framework)
+
+{insert description}
```
Comment (emilyrohrbough) [inline packages/terra-application-utility/README.md]: Add description

File: packages/terra-application-utility/docs/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Terra Application Utility
+
+{insert description}
```
Comment (emilyrohrbough) [inline packages/terra-application-utility/docs/README.md]: Add description

File: packages/terra-application-utility/docs/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-application-utility`
+
+## Usage
+
+```jsx
+import React from 'react';
+import ApplicationUtility from 'terra-application-utility';
+
+<ApplicationUtility {props...} />
```
Comment (emilyrohrbough) [inline packages/terra-application-utility/docs/README.md]: Update with true example

File: packages/terra-application-utility/docs/README.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+```
+
+## Component Features
+
+<!-- Uncomment supported features.
```
Comment (emilyrohrbough) [inline packages/terra-application-utility/docs/README.md]: Add supported features

File: packages/terra-application-utility/package.json [hunk 5]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "terra-application-utility",
+  "main": "lib/ApplicationUtility.js",
+  "version": "0.0.0",
+  "description": "{insert description}",
```
Comment (emilyrohrbough) [inline packages/terra-application-utility/package.json]: Add description

**Comment by emilyrohrbough** [comment]
Started to review the code and realized this is the scaffolding 😅

**Comment by yuderekyu** [comment]
@emilyrohrbough 👎

**Pull request merged**